### PR TITLE
Create delivery address popup

### PR DESCRIPTION
**Pull request by mplayer78**
basic address detail popup added.

### CODE REVIEW

File: chowpronto_frontend/src/hooks/useAuth.js [hunk 1]
```diff
@@ -37,25 +45,28 @@ function useAuth() {
...
     window.localStorage.removeItem("chowpronto");
+    setUser({ type: "set_user", userDetails: {} });
   }
 
   function register(customerDetailsObject) {}
   function setToStorage(dataObj) {
     try {
-      window.localStorage.setItem("chowpronto", JSON.stringify(dataObj.token));
+      window.localStorage.setItem("chowpronto", dataObj.token);
+      console.log(window.localStorage.getItem("chowpronto"));
```
Comment (kosenchiha) [inline chowpronto_frontend/src/hooks/useAuth.js:61]: console.log should be removed :)

File: chowpronto_frontend/src/shared_components/CloseIcon.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+function Icon() {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      // width="25"
+      // height="25"
```
Comment (kosenchiha) [inline chowpronto_frontend/src/shared_components/CloseIcon.js:8]: We need to start tidying up our code and remove all comments

File: chowpronto_frontend/src/hooks/useAuth.js (hunk 1 above)
Comment (mplayer78) [reply chowpronto_frontend/src/hooks/useAuth.js]: done

File: chowpronto_frontend/src/shared_components/CloseIcon.js (hunk 2 above)
Comment (mplayer78) [reply chowpronto_frontend/src/shared_components/CloseIcon.js:8]: will create a new PR for this

**Pull request merged**